Taarafo.Core/Services/Foundations/Posts/PostService.Validations.cs

```diff
@@ -39,6 +41,14 @@ private static void ValidatePostIsNotNull(Post post)
             }
         }
 
+        public void ValiateStoragePost(Guid postId, Post maybePost)
```

Comment:
Switch your input parameters based on usage.
We check on `maybePost` first before we use the `postId` let's honor that order in the input params please.